### PR TITLE
[TW-1358] Add team members to Enterprise Essentials solutions

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
Explain how to add team members to Enterprise Essentials solutions, and explain the impact of adding users to a solution.

**Beta site:**
* [Manage your team's access to Postbot](https://tw-1358-enterprise-essentials-add-users.learning.postman-beta.com/docs/administration/managing-your-team/managing-your-team/#manage-your-teams-access-to-postbot)
* [Manage your team's access to Enterprise solutions](https://tw-1358-enterprise-essentials-add-users.learning.postman-beta.com/docs/administration/managing-your-team/managing-your-team/#manage-your-teams-access-to-enterprise-solutions)
* [Purchasing add-on products](https://tw-1358-enterprise-essentials-add-users.learning.postman-beta.com/docs/billing/billing/#purchasing-add-on-products)
* [Manage Enterprise solutions licenses](https://tw-1358-enterprise-essentials-add-users.learning.postman-beta.com/docs/billing/billing/#manage-enterprise-solutions-licenses)
* [Postbot calls](https://tw-1358-enterprise-essentials-add-users.learning.postman-beta.com/docs/billing/resource-usage/#postbot-calls)

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 😄

**Review by malvikach — CHANGES_REQUESTED** [review]

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 1]
```diff
@@ -218,6 +221,46 @@ To give all team members permission to create team workspaces, do the following:
...
+
+When you add team members to the Internal API Management solution:
+
+* You can then assign them the [API Network Manager and API Governance Manager](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) roles. Learn more about [managing team roles](#managing-team-roles).
+* All team members get access to the Private API Network.
+* Only team members added to this solution can be assigned the [API Editor](/docs/collaborating-in-postman/roles-and-permissions/#api-roles) role in Postman APIs. Team members not added to this solution will only have the [API Viewer](/docs/collaborating-in-postman/roles-and-permissions/#api-roles) role.
+
+### API Test Automation solution
+
+You can add team members to the API Test Automation solution for more access to the [Collection Runner](/docs/collections/running-collections/intro-to-collection-runs/), [CI/CD integrations](/docs/integrations/ci-integrations/), [Postman Monitors](/docs/monitoring-your-api/intro-monitors/), and more.
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md]: This is now updated.

File: src/pages/docs/billing/billing.md [hunk 2]
```diff
@@ -263,22 +263,23 @@ In your [billing dashboard](http://go.postman.co/billing), you can view:
 
 > [Contact Postman support](https://www.postman.com/support/) with any questions about Auto-Flex.
 
-## Managing solution access
+## Manage Enterprise solutions licenses
 
-> **[Enterprise Solutions are available for Enterprise Essentials plans.](https://www.postman.com/pricing)**
+> **[Enterprise solutions are available for Enterprise Essentials plans.](https://www.postman.com/pricing)**
```
Comment (taehoshino) [inline src/pages/docs/billing/billing.md:268]: I would suggest adding this section immediately after "Purchasing add-on products" section (that is to be added as part of https://github.com/postmanlabs/postman-docs/pull/5136

**Comment by taehoshino** [comment]
@jkonrath-postman Are we also adding a section to explain how to manage Postbot access from Product Access page?

**Comment by MatthewPinsker-postman** [comment]
> @jkonrath-postman Are we also adding a section to explain how to manage Postbot access from Product Access page?

Hi @taehoshino! Are we adding Postbot to the Product access page in the October release?

File: src/pages/docs/billing/billing.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/billing/billing.md:268]: Thanks! Once https://github.com/postmanlabs/postman-docs/pull/5136 is merged with the `develop` branch, I'll pull the changes into this branch and look at updating/reorganizing the Enterprise solutions content.

**Comment by jkonrath-postman** [comment]
> @jkonrath-postman Are we also adding a section to explain how to manage Postbot access from Product Access page?

We can after we ship on Oct 16. This PR is going to have to get the changes from the other one merged into it first. (@MatthewPinsker-postman this will make more sense after you merge develop back into here on Monday.)

**Comment by taehoshino** [comment]
> > @jkonrath-postman Are we also adding a section to explain how to manage Postbot access from Product Access page?
> 
> Hi @taehoshino! Are we adding Postbot to the Product access page in the October release?

Yes.

**Review by shashankawasthi88 — APPROVED** [review]
looks good

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 3]
```diff
@@ -218,6 +221,68 @@ To give all team members permission to create team workspaces, do the following:
 
     <img alt="All team members can create team workspaces" src="https://assets.postman.com/postman-docs/v10/workspace-management-all-team-members-v10.jpg" width="600px"/>
 
+## Manage your team's product access
+
+> **[Product access is available for Enterprise Essentials plans.](https://www.postman.com/pricing)**
```
Comment (taehoshino) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:226]: Postbot section is available for Basic and Prof as well

**Comment by MatthewPinsker-postman** [comment]
@taehoshino asked me to include Shivam Sharma in this review. Shivam reviewed the Postbot-related changes and approved them via Slack on 10/19/2023.

**Pull request merged**